packages/near-fast-auth-signer-e2e-tests/firebase.ts

```diff
@@ -0,0 +1,74 @@
+import admin from 'firebase-admin';
+
+import { serviceAccount } from './utils/serviceAccount';
+
+admin.initializeApp({
+  credential: admin.credential.cert(serviceAccount as admin.ServiceAccount)
+});
+
+interface FirebaseAuthResponse {
+  idToken: string;
+  refreshToken: string;
+  expiresIn: string;
+  localId: string;
+}
+
+async function createNewUser(email: string, password: string): Promise<admin.auth.UserRecord> {
+  try {
+    const userRecord = await admin.auth().createUser({
+      email,
+      password
+    });
+    return userRecord;
+  } catch (error) {
+    console.error('Error creating new user:', error);
+    throw error;
+  }
+}
+
+async function signInAndGetToken(email: string, password: string): Promise<string> {
+  const apiKey = 'AIzaSyDAh6lSSkEbpRekkGYdDM5jazV6IQnIZFU';
+  if (!apiKey) {
+    throw new Error('Firebase API key is missing');
+  }
```

Comment:
either make API key configurable or remove this error. (I think just remove if statement is more valid..)